content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+
+```{r echo=FALSE, out.width="49%", fig.link="https://github.com/allisonhorst/stats-illustrations", fig.cap="Art by Allison Horst", fig.align='center'}
+knitr::include_graphics("parsnip.png")
+```
+
+We designed the workshop to provide a gentle introduction to machine learning and to the tidyverse packages that do machine learning. Attendees learned how to train and assess predictive models with several common machine learning algorithms, as well as how to do feature engineering to improve the predictive accuracy of their models. We focused on learning the basic theory and best practices that support machine learning. Along the way, we introduced several core tidymodels packages including [parsnip][parsnip], [recipes][recipes], [rsample][rsample], and [tune][tune], which provide a grammar for modeling and work seamlessly with R's tidyverse packages. 
+
+```{r echo=FALSE, out.width="49%", fig.link="https://github.com/allisonhorst/stats-illustrations", fig.cap="Art by Allison Horst", fig.align='center'}
+knitr::include_graphics("recipes.png")
+```
+
+## Is this course for me?
+
+Before workshops for this year's conf were announced, we framed two questions to help potential learners gauge whether this workshop was the right one for them:
+
+---
+
++ Can you use mutate and purrr to transform a data frame that contains list columns?
+
++ Can you use the ggplot2 package to make a large variety of graphs?
+
+If you answered "no" to either question, you can brush up on these topics by working through the online tutorials at https://rstudio.cloud/learn/primers.
+
+---
+
+These questions were driven by the fact that when we started developing the workshop, tidymodels code heavily relied on the purrr package (see an end-to-end code example from Max's Applied Machine Learning workshop at rstudio::conf(2019) [here](https://github.com/topepo/rstudio-conf-2019/blob/2aaf4c24cd90cfa91cf2cdde256d68341f21133b/Materials/Part_2_Basic_Principles.R#L137-L212)). However, between the time we first conceived of the workshop and when we taught it, a lot of the tidymodels API had changed (for the better!). In hindsight, here is the minimum tidyverse comfort level you need to be able to dive into tidymodels:
+
++ Can you use the [pipe operator](https://r4ds.had.co.nz/transform.html#combining-multiple-operations-with-the-pipe) to combine a sequence of functions to transform objects in R (like a data frame)?
+
++ Can you work with tibbles (or data frames) that contain [list columns](https://r4ds.had.co.nz/many-models.html#list-columns-1)?
+
++ Can you use [`dplyr::select()` helper functions](https://tidyselect.r-lib.org/reference/select_helpers.html)? (this helps a lot when you get to the [recipes][recipes] package)
```

Comment:
Parenthetical phrase at the end of the line is dangling: please incorporate it into the sentence or make it one in its own right.